packages/ketcher-react/src/script/ui/dialog/template/TemplateTable.tsx

```diff
@@ -39,23 +40,22 @@ export interface Template {
   }
 }
 
+const getSettingsSelector = state => state.options.settings
+
 function tmplName(tmpl: Template, i: number): string {
   return tmpl.struct.name || `${tmpl.props.group} template ${i + 1}`
 }
 
 const RenderTmpl: FC<{
   tmpl: Template
+  options: any
```

Comment:
We are not using 'any' type

Reply:
It can be fixed later: react components library will be reworked in scope of 2.5